### PR TITLE
Swiftgen and Clean up Localizable.strings

### PR DESCRIPTION
**Pull request by PangMo5**
I apply Swiftgen for the new localization system.

The benefits written in Swiftgen repo.
- Avoid any risk of typo when using a String
- Free auto-completion
- Avoid the risk of using a non-existing asset name
- All this will be ensured by the compiler and thus avoid the risk of crashing at runtime.

Before
```swift
Text("Home")
Text("S\(String(item.parentIndexNumber ?? 0)):E\(String(item.indexNumber ?? 0))")
```

After
```swift
L10n.home.text
Text(L10n.seasonAndEpisode(item.parentIndexNumber ?? 0, item.indexNumber ?? 0))
```

For more information, see https://github.com/SwiftGen/SwiftGen.

- [x] Apply Swiftgen
- [x] Remove some unnecessary keys (ex: `%@ • %@`, `Genres:`)
- [x] Change all the string usage parts

If you have any additional opinions on localization, please give me.

I didn't touch the string, which was not translated before.

### CODE REVIEW

**Comment by PangMo5** [comment]
It is recommended that this PR be merged before localization is updated due to the Weblate.

**Review by LePips — DISMISSED** [review]

**Comment by LePips** [comment]
Just as dev note: I had trouble with CocoaPods since I'm now using an M1 machine and had to run some commands from this thread: https://github.com/CocoaPods/CocoaPods/issues/10220

I will document this

**Comment by LePips** [comment]
Also, the generated file has swiftlint ignore rules. Because of this I'm almost convinced to just scrap swiftformat and keep to swiftlint

**Comment by acvigue** [comment]
Is this PR ready to merge?

**Comment by LePips** [comment]
It should be, yes

File: JellyfinPlayer tvOS/Views/VideoPlayer/AudioView.swift [hunk 1]
```diff
@@ -14,7 +14,7 @@ class AudioViewController: InfoTabViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        tabBarItem.title = NSLocalizedString("Audio", comment: "")
+        tabBarItem.title = "Audio"
```
[the remark this answers is not in the document]
Comment (PangMo5) [reply JellyfinPlayer tvOS/Views/VideoPlayer/AudioView.swift:17]: This part was confirmed, but there was no key related to "Audio", so it was processed as a Plain String for now.

**Comment by PangMo5** [comment]
> Also, the generated file has swiftlint ignore rules. Because of this I'm almost convinced to just scrap swiftformat and keep to swiftlint

Oh, this is because I used the default template provided by Swiftgen.
You can also change it to SwiftFormat ignore rules if you want.
Is there any other reason to stop changing to SwiftFormat?

**Comment by LePips** [comment]
> Is there any other reason to stop changing to SwiftFormat?

Oh no, I had just saw that I thought it would be a problem. Since it isn't, we're still good to go on SwiftFormat.

**Comment by PangMo5** [comment]
> Is this PR ready to merge?

Now it's ready 😄

**Review by acvigue — APPROVED** [review]

**Pull request merged**